recipes/example-robot-data/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+cmake .. \
+      -DCMAKE_PREFIX_PATH=$PREFIX \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      -DPYTHON_EXECUTABLE=$PYTHON
+
+make install
```

Comment:
Probably should add `-j {CPU_COUNT}`

Reply:
There is no compilation, so this is not useful.